javascript/lerna.json

```diff
@@ -16,5 +16,5 @@
       ]
     }
   },
-  "version": "2.0.0"
+  "version": "2.1.0-alpha.1"
```

Comment:
Version 2.0.0 is not released. Consider the master branch the release candidate of 2.0.0. Please leave the version as-is.

Reply:
Okay, no problem, allthough I'm a bit confused as the current version on NPM is tagged as 2.0.0 without any indication that it's a release-candidate/pre-release: https://www.npmjs.com/package/@eclipse-ditto/ditto-javascript-client-api_1.0

Reply:
You're right, 2.0.0 is already released, so you could put 2.1.0 here.

Reply:
Well, now I already rolled back the version number to 2.0.0 😬 
I hope that's okay as well.

Reply:
Yes, that is fine.